R/pyramid_chart.R

```diff
@@ -0,0 +1,99 @@
+#' Demography Figure
+#'
+#' @param fig2_data `dataframe` demography data
```

Comment:
We may consider avoiding the use of figX in the codes.

Reply:
Agreed - we can discuss further.